src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Instead, set styles only for elements that require them. If you need to apply a global box-sizing, consider using `html` selector.